components/checkbox/index.css

```diff
@@ -200,17 +209,17 @@ governing permissions and limitations under the License.
   opacity: 0;
   transform: scale(0);
 
-  transition: opacity var(--spectrum-global-animation-duration-100) ease-in-out, transform var(--spectrum-global-animation-duration-100) ease-in-out;
+  transition: opacity var(--spectrum-checkbox-animation-duration) ease-in-out, transform var(--spectrum-checkbox-animation-duration) ease-in-out;
 }
 
 .spectrum-Checkbox-checkmark {
-  margin-block-start: calc(var(--spectrum-checkbox-checkmark-size) / -2);
-  margin-inline-start: calc(var(--spectrum-checkbox-checkmark-size) / -2);
+  /* margin-block-start: calc(var(--spectrum-checkbox-checkmark-size) / -2); */
+  /* margin-inline-start: calc(var(--spectrum-checkbox-checkmark-size) / -2); */
```

Comment:
How do these icons work? I see the t-shirt sizes in the XD file, but they don't have token names.

<img width="711" alt="Screen Shot 2022-06-30 at 8 28 06 AM" src="https://user-images.githubusercontent.com/16216104/176562494-b969f67a-7940-44fb-95ea-48538b61431e.png">

Reply:
See https://github.com/adobe/spectrum-css/pull/1465#discussion_r912085022.